x/programs/cmd/simulator/src/lib.rs

```diff
@@ -3,17 +3,15 @@
 //! Alternatively the `Plan` can be written in JSON and passed to the
 //! Simulator binary directly.
 
+use serde::{Deserialize, Serialize};
```

Comment:
Write comments `lib.rs` (here) describing that this file is used to wrap the Go cli-tool